py/pykms/pykmsutil.cpp

```diff
@@ -26,10 +26,13 @@ void init_pykmstest(py::module &m)
 			.def("reserve_connector", (Connector* (ResourceManager::*)(const string& name))&ResourceManager::reserve_connector,
 			     py::arg("name") = string())
 			.def("reserve_crtc", &ResourceManager::reserve_crtc)
-			.def("reserve_plane", &ResourceManager::reserve_plane,
+			.def("reserve_plane", (Plane* (ResourceManager::*)(Crtc*, Plane::Type, PixelFormat)) &ResourceManager::reserve_plane,
 			     py::arg("crtc"),
```

Comment:
You changed these functions in the previous patch, but only fix it here for py. Doesn't that break compilation if you compile the prev commit?